templates/Feature.js

```diff
@@ -0,0 +1,78 @@
+const fs = require("fs");
+const globby = require("globby");
+const mustache = require("mustache");
+
+const template = `const HFX = require("../../HFX");
+
+class {{{ name }}} extends HFX.Feature {
+  constructor() {
+    super({
+      section: HFX.Section.{{{ section }}},
+      name: "{{{ nameSpaced }}}",
```

Comment:
Is this suppose to be nameSpace**d**?

Reply:
Yeah, its the spaced-out name. So the feature name (class) `SomeNewFeature` becomes `Some New Feature`.